src/tab-group.ts

```diff
@@ -270,6 +288,7 @@ export default class TabGroup extends LitElement {
 
   #onDefaultSlotChange() {
     this.#setAriaAttributes();
+    this.#validateTabPanelPairs();
```

Comment:
Panels are added to the default slot, so we need to validate them when it changes.